onadata/libs/authentication.py

```diff
@@ -56,16 +63,26 @@ def build_challenge_response(self):
         return self.authenticator.build_challenge_response()
 
 
-class HttpsOnlyBasicAuthentication(BasicAuthentication):
+class HttpsOnlyBasicAuthentication(MFABlockerMixin, BasicAuthentication):
+    """
+    Extend DRF class to support MFA and authentication over HTTPS only (if
+    testing mode is not activated)
+
+    Basic authentication should be deactivated if user has activated MFA
+    on their account (unless it has been add to `settings.MFA_SUPPORTED_AUTH_CLASSES`)
+    """
+    verbose_name = 'Https Basic authentication'
```

Comment:
What do these get used for? I poked around the DRF code but didn't figure it out

Reply:
This is used by the AuthenticationFailed exception to display an explicit error. 
`verbose_name` seems to be a variable name often used in Django (e.g. Django Model Meta class, AppConfig etc...)